build/LegacyPackages/14.0/Microsoft.Build.Framework/Microsoft.Build.Framework.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
```

Comment:
Are these `minClientVersion`s correct? Doesn't using netstandard1.3 raise the bar?